lib/src/csv_asset_loader.dart

```diff
@@ -12,6 +12,9 @@ import 'asset_loader.dart';
 //
 class CsvAssetLoader extends AssetLoader {
   CSVParser? csvParser;
+  final Locale? fallbackLocale;
+
+  CsvAssetLoader([this.fallbackLocale]);
```

Comment:
I would prefer optional args over this:

`CsvAssetLoader({this.fallbackLocale});`